ska_service/ska_service/ska/base.py

```diff
@@ -0,0 +1,65 @@
+"""Ska2 base command."""
+
+import logging
+import subprocess
+from pathlib import Path
+from subprocess import CalledProcessError, CompletedProcess
+from typing import Sequence
+
+LOG = logging.getLogger(__name__)
+
+
+def _format_cli_options(options: dict[str, str | int | Path]) -> Sequence[str]:
```

Comment:
Why not using `argparse` (or `click`) for this?

Reply:
OK, looks like I was reading too quickly. You are building for subprocess here. It does not make sense to use libraries here.